dspace/etc/postgres/update-sequences.sql

```diff
@@ -47,38 +43,21 @@
 -- org.dspace.content API which is safe to use concurrently and in multiple
 -- JVMs.  The SQL code below will typically only be required after a direct
 -- SQL data dump from a backup or somesuch.
- 
+
 
 -- There should be one of these calls for every ID sequence defined in
 -- database_schema.sql.
 
 
 SELECT setval('bitstreamformatregistry_seq', max(bitstream_format_id)) FROM bitstreamformatregistry;
 SELECT setval('fileextension_seq', max(file_extension_id)) FROM fileextension;
-SELECT setval('bitstream_seq', max(bitstream_id)) FROM bitstream;
-SELECT setval('eperson_seq', max(eperson_id)) FROM eperson;
-SELECT setval('epersongroup_seq', max(eperson_group_id)) FROM epersongroup;
-SELECT setval('group2group_seq', max(id)) FROM group2group;
-SELECT setval('group2groupcache_seq', max(id)) FROM group2groupcache;
-SELECT setval('item_seq', max(item_id)) FROM item;
-SELECT setval('bundle_seq', max(bundle_id)) FROM bundle;
-SELECT setval('item2bundle_seq', max(id)) FROM item2bundle;
-SELECT setval('bundle2bitstream_seq', max(id)) FROM bundle2bitstream;
-SELECT setval('dcvalue_seq', max(dc_value_id)) FROM dcvalue;
-SELECT setval('community_seq', max(community_id)) FROM community;
-SELECT setval('community2community_seq', max(id)) FROM community2community;
-SELECT setval('collection_seq', max(collection_id)) FROM collection;
-SELECT setval('community2collection_seq', max(id)) FROM community2collection;
-SELECT setval('collection2item_seq', max(id)) FROM collection2item;
 SELECT setval('resourcepolicy_seq', max(policy_id)) FROM resourcepolicy;
-SELECT setval('epersongroup2eperson_seq', max(id)) FROM epersongroup2eperson;
 SELECT setval('workspaceitem_seq', max(workspace_item_id)) FROM workspaceitem;
 SELECT setval('workflowitem_seq', max(workflow_id)) FROM workflowitem;
 SELECT setval('tasklistitem_seq', max(tasklist_id)) FROM tasklistitem;
 SELECT setval('registrationdata_seq', max(registrationdata_id)) FROM registrationdata;
 SELECT setval('subscription_seq', max(subscription_id)) FROM subscription;
 SELECT setval('communities2item_seq', max(id)) FROM communities2item;
```

Comment:
This sequence has already been dropped in another 6.0 migration. So it can be removed from both update scripts too
https://github.com/DSpace/DSpace/blob/master/dspace-api/src/main/java/org/dspace/storage/rdbms/migration/V6_0_2016_01_26__DS_2188_Remove_DBMS_Browse_Tables.java#L141